src/content/docs/Company/ceo.md

```diff
@@ -29,23 +27,26 @@ Prior to Google Eric lead a small startup for 15 months as part of
 Y-Combinator's Summer 2006 batch, and prior to that worked on Safari at Apple
 where he brought SVG to Safari and helped start the Safari on Windows project.
 
+### Talks
+
+- [Flutter at 10 years, FlutterCon, Sept 2024](https://www.youtube.com/watch?v=DgEplfkrYWQ)
```

Comment:
Can add your GCN talk here if you would like https://www.youtube.com/watch?v=h1CJoauhPaE

Reply:
Done.